### PR TITLE
MAUI-3588 How-to-create-a-dotted-line-chart-in-.NET-MAUI-Chart

### PR DESCRIPTION
**Pull request by RajaRamalingam3772**
Added the files of the dotted line chart sample.

### CODE REVIEW

File: DottedLinechartSample/DottedLinechartSample/DottedLinechartSample.csproj [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		<!-- Custom Fonts -->
+		<MauiFont Include="Resources\Fonts\*" />
+
+		<!-- Raw Assets (also remove the "Resources\Raw" prefix) -->
+		<MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <PackageReference Include="Syncfusion.Maui.Charts" Version="20.3.58" />
+	</ItemGroup>
```
Comment (MuneeshKumarG) [inline DottedLinechartSample/DottedLinechartSample/DottedLinechartSample.csproj:53]: @RajaRamalingam3772 : Mention version as *.

**Comment by MuneeshKumarG** [comment]
@RajaRamalingam3772 : ReadMe file content is missing.

**Comment by RajaRamalingam3772** [comment]
> @RajaRamalingam3772 : ReadMe file content is missing.

Hi @MuneeshKumarG sir, I have updated the read me content.

File: DottedLinechartSample/DottedLinechartSample/DottedLinechartSample.csproj (hunk 1 above)
Comment (RajaRamalingam3772) [reply DottedLinechartSample/DottedLinechartSample/DottedLinechartSample.csproj:53]: Hi @MuneeshKumarG sir, now I have mentioned the version as '*'. Please verify it.

**Review by MuneeshKumarG — APPROVED** [review]
This is fine

**Pull request merged**